bsp_diff/caas/device/intel/civ/host/vm-manager/0043-Add-parameters-to-config-ini-to-support-HPD-feature.patch

```diff
@@ -0,0 +1,45 @@
+From 8a7b95554716dd9f981697f70882724012de559a Mon Sep 17 00:00:00 2001
+From: "Suresh, Prashanth" <prashanth.suresh@intel.com>
+Date: Wed, 5 Apr 2023 14:19:14 +0530
+Subject: [PATCH] Add parameters to config ini to support HPD feature.
+
+Add display related featured for HPD support on Android.
+Tracked-On:
+---
+ sample/guest01.ini    | 7 +++++++
+ scripts/setup_host.sh | 1 +
+ 2 files changed, 8 insertions(+)
+
+diff --git a/sample/guest01.ini b/sample/guest01.ini
+index 0542a20..5349fbd 100644
+--- a/sample/guest01.ini
++++ b/sample/guest01.ini
+@@ -71,6 +71,13 @@ support_suspend=enable
+ #making audio passthrough for guest. In all other cases use "legacy-hda" only.
+ audio_type=legacy-hda
+ 
++#"show_fps" should be set to "on" if FPS is to be displayed. It can be set to "off" or left blank
++#if not required. Any other value will result in FPS being off.
```

Comment:
I think we should only have the following:
- commented out show_fps
- show_fps=on
- show_fps=off

Any other value should be an error.